data/avro-schema/src/main/resources/avro/net/corda/data/p2p/mtls/AllClientCertificateSubjects.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "record",
+  "name": "AllClientCertificateSubjects",
```

Comment:
I'd suggest we move that into `net.corda.data.p2p.mtls.gateway` namespace/package and then rename the class into `AllowedCertificateSubject`. I suppose the clash is the reason that prevented you from using the same name..

Reply:
also, let's add a doc at the root level with some docs for the class, e.g. `Certificate subject to allow TLS connections from by the gateway`.

Reply:
I renamed all three of them so the names will be more explicit...